src/Features/CSharp/Portable/Completion/SuggestionMode/CSharpSuggestionModeCompletionProvider.cs

```diff
@@ -91,6 +93,20 @@ private bool IsAnonymousObjectCreation(SyntaxToken token)
             return false;
         }
 
+        private bool IsPossibleTupleExpression(SyntaxToken token)
```

Comment:
I think we may be able to make this smarter, but for now I will keep the broader behavior.